examples/testapp/src/pages/add-address/components/AddAddress.tsx

```diff
@@ -15,6 +15,7 @@ export function AddAddress({ sdk }: { sdk: ReturnType<typeof createCoinbaseWalle
       method: 'wallet_addAddress',
       params: [
         {
+          version: '1',
```

Comment:
thx! i should also do a  fallback on FE

Reply:
I can add that in my PR that fixes the schema. Should be ready here in a few. Updating some tests now